client/src/components/Calendar/index.js

```diff
@@ -51,6 +51,13 @@ class BigCalendar extends Component {
     history.push({ pathname: "/detailsevent", event });
   };
 
+  pendingEventStyle = (event, start, end, isSelected) => {
```

Comment:
Why did you define ```start```, ```end```, and ```isSelected```! you didn't use it in the function!

Reply:
removed